### ISSUE TITLE
Address not appearing as a shdwNode even tho every step done

### ISSUE THREAD

**Issue by TsolakidisK** [GenesysGo/dagger-testnet-issues#27]
**Describe the bug**
Since 0.8.0 has been released from the first minute i upgraded my wizards version and run the node on my VPS.
I have read the complete documentation several times (getting private keys / words, importing to a solana wallet, connecting my discord, etc.) I did this since yesterday 3 times with new keys but i am still not able to see my shdwNode as a node and either i can see my node in the leaderboard.

meanwhile after doing this several times i am convinced that this has to be a "bug"

a cNFT has also not appeared in my wallet yet.

i will add some screenshots that verify, that i have done everything as supposed and i would appreciate if some1 can help me out.
Discord is so crowded and i couldnt get any help until now.

<img width="751" alt="Screenshot 2024-01-17 at 11 11 31" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/39744422/d61fa4d0-9ce5-4a28-a4eb-46c886369ee3">
<img width="1002" alt="Screenshot 2024-01-17 at 11 11 54" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/39744422/96d79de7-3d21-4cd4-a4e2-582c636048f6">
<img width="868" alt="Screenshot 2024-01-17 at 11 12 09" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/39744422/90f29b4f-7db7-4ed9-b901-bd761bd0f7c3">

**Comment by Cre8tivPrime8** [comment]
Also experiencing this: Unable to get Node ID recognized for discord verification with a previously spun up node that had been tracked before (Updated to latest version with at least 100 $SHDW in node wallet and staking wallet and finalizing in logs)
Is shdwNode shows as "No" and no cNFTs received: 
Node ID: 6Fv9bYXVK6HgspzJye9Ptu9ZhbayGwwYHC63UzfAHRY
Staking Wallet: 9N8KBLzrnLCPckoyEoin5zmrBnKYcHUQUPy6oaJxGDgJ

**Comment by Zoturi** [comment]
Same issue. 
- Can't get the node account recognized as shadow node, even though I can see that I've connected the AwEG..account to my discord ID. Also added my holding account.
- Can't see my node in the global dashboard
- Can't see my node on the Node Operators Reward screen (just a blank screen), even though I did see it on the dashboard during phase 1 (spun my node up just a day or two before the end of phase one, so didn't participate, just checked that everything looks ok)
- Staking page shows account correctly

Node ID: AwEGxFZ49bCqDP2NQFC9GRJZRHnKshPrrzHinWFJcrmJ

I've also added (later on) the account that is derived from the rec phrase, just in case.

`dagger@ip-10-7-1-59:~$ sudo tail -f config.log ø grep -i warn
2024-01-17T12:04:18.634635584+00:00 ÆWARNÅ dagger_logger::metrics - run: submit status: 204
2024-01-17T12:04:29.295735584+00:00 ÆWARNÅ dagger_logger::metrics - run: submit status: 204
^C
dagger@ip-10-7-1-59:~$ sudo tail -f config.log ø grep -i finalized
2024-01-17T12:04:47.209434074+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87168 finalized!
2024-01-17T12:04:48.803366623+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87169 finalized!
2024-01-17T12:04:48.803408314+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87170 finalized!
2024-01-17T12:04:49.581849045+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87171 finalized!
2024-01-17T12:04:50.640659434+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87172 finalized!
2024-01-17T12:04:51.445207905+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87173 finalized!
2024-01-17T12:04:51.445242826+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87174 finalized!
2024-01-17T12:04:52.394438671+00:00 ÆINFOÅ dagger::p2p::graph::dagger - bundle 87175 finalized!
^C
dagger@ip-10-7-1-59:~$ ./wield --version
wield 0.8.0`

![verification](https://github.com/GenesysGo/dagger-testnet-issues/assets/4124306/46eab840-5e0b-45f7-85c7-7fecb0c08617)

![reward](https://github.com/GenesysGo/dagger-testnet-issues/assets/4124306/bbb5b85e-2bb4-441b-90ac-e13265afcd36)
![stake](https://github.com/GenesysGo/dagger-testnet-issues/assets/4124306/acb68ac5-21f5-4c03-b12e-cf43311fb7bc)


Some troubleshooting steps I've taken:
- Originally I used Solflare as the wallet app, but I also tried to add the same account (with priv key from node, so the correct account) to Phantom and currently the account is attached using Phantom. 
- I haven't restarted the node, to keep my uptime and as the server is doing Daggery things.
- I've reattached my node acco to the verification page multiple times.

**Comment by rumnkoke** [comment]
Same issue here. Node iD:FpXnRmLkeKpR3d21aDe1fRZXHehSe1YbwG59XyPxjm6b

**Comment by ajayp14** [comment]
same issue

**Comment by HavaneFr** [comment]
same issue : 
- same node ID and wallet adress
- more than 100 shdw (was 500 at the beginning....)
- finalized in the log
- vers 0.8.0 since 1 min after the annoucement
- discord verification page, show connected with the wallet above : tried disconnected, reconnected. No change : no role added

First it was because of solana congestion and fee. But hours later, it's not that

**Comment by grubbins2** [comment]
Same issue with two nodes:
* `CPY4itR3pDyXLCqAYnhhcXGqaNccMGs48oFAB741tLEj`
* `HfBUoTqvnM5FmMZEGTbdpQrD6V1Sz6Rf6NrQZ6Ev8RHW`

They are both finalizing just fine. I did import the private key into Phantom per the instructions. 

<img width="762" alt="Screenshot 2024-01-17 at 5 21 02 AM" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/13593901/920dfdac-337c-4ad7-9859-4dc6a5e72a36">

**Comment by fetith** [comment]
same issue.
- same node ID and wallet adress
- more than 100 shdw (was 500 at the beginning....)
- finalized in the log
- vers 0.8.0 since 1 min after the annoucement
- discord verification page, show connected with the wallet above : tried disconnected, reconnected. No change : no role added

**Comment by sweeptoshi** [comment]
Same for me:
- Same node id and wallet address: FUwGRcnBFmKWJwn2BqAQtoRvZhQZKUVphj1W3yFKEcot
- 500 and change $shdw in wallet
- node is running and finalizing 

Wallet connected to discord verification, deleted, connected again. Also tried restarting the node. 
still showing “is shdwNode” no and no cnft in wallet. 

Discord id: sweeptoshi#4780
![IMG_6595](https://github.com/GenesysGo/dagger-testnet-issues/assets/156907755/eab8762a-3d78-4d6b-a110-570a46b320e9)
![IMG_6594](https://github.com/GenesysGo/dagger-testnet-issues/assets/156907755/3ac8a811-7dba-4706-8c2f-cd8d9bcf2437)

**Comment by beardy46678** [comment]
Same issue here as well. Never showed status before or had NFT before as just started with testnet2 - so would be a new node/NFT/etc.

Node ID and Wallet Address: C6A4tcihc6i62M2ZYxgYYmRwc6smGXL1tAuYoZkrmmdb
Discord id: beardy4660

- Showing as all finalized when running `tail -f config.log | grep "finalized"`
- Wield is running
- Uptime 12.3 hours according to Grafana dashboard.
- Have 501 SHDW in node wallet, as well as 500 SHDW staked in node wallet.

Showing as Is shdwNode=No and not seen on any node operator lists or leaderboards, even queued. 
<img width="821" alt="Screenshot 2024-01-17 at 10 30 41 AM" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/156930792/4543eaf9-3c96-45e4-b24e-a2cb7eec3281">

**Comment by beardy46678** [comment]
Update that might help others. TL;DR - Reinstall node using installer script.
 
I realized I had regenerated my keypair/priv key on my node because I was having trouble adding my wallet to Solana using the Private Key - since we've discovered that using the Recovery Phrase doesn't work right and doesn't generate the wallet address as the Node ID correctly.

After checking the public node list via grafana and downloading the CSV (https://dashboard.shdwdrive.com/d/b14b5606-fb9c-4a2a-84fc-80887f144965/dagger-public-data?orgId=1&refresh=30m&viewPanel=11&inspect=11) - I saw my **old** node ID was on the list, but not the new one.

I went into my node, stopped the Wield service, waited 5-6 epochs, started Wield back up - the node started showing up in the public node list and I'm showing as Is shdwNode=Yes in Discord verify after disconnecting and reconnecting my wallet.

Did notice that afterwards I wasn’t getting finalized - if I removed grep “finalized” and just checked `tail -f config.log` I would get:

> 2024-01-18T18:33:51.660813118+01:00 [WARN] dagger_logger::metrics - run: submit status: 204
> 2024-01-18T18:34:06.128887767+01:00 [INFO] dagger_logger::metrics - datapoint: uptime_metrics node_id="C6A4tcihc6i62M2ZYxgYYmRwc6smGXL1tAuYoZkrmmdb" is_up=false start_ts=0 current_uptime_ms=0 uptime_added_ms=0 last_successful_sync_ts=0


Re-ran the node installer script, uninstalled, reinstalled, and showed as finalizing again.

NFT arrived in wallet.

Now only problem is it's showing **Current status: Offline** when checking my Node Operator Rewards page.

<img width="513" alt="Screenshot 2024-01-18 at 12 56 32 PM" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/156930792/028d7d28-b58e-4d72-8616-d4b20a007b95">
